test/reagent/cookies_test.cljs

```diff
@@ -1,6 +1,22 @@
 (ns reagent.cookies-test
-  (:require [cljs.test :refer-macros [deftest testing is]]
-            [reagent.cookies :as cookies]))
+  (:require
+    [cljs.test :refer-macros [deftest testing is use-fixtures]]
+    [clojure.string :as str]
+    [reagent.cookies :as cookies]))
+
+(defn- stub-cookies [f]
+  (let [old (.getInstance goog.net.Cookies)
+        stub (new goog.net.Cookies nil)]
+    (set! goog.net.cookies stub) ;NOTE the google closure library marks this method as deprecated
```

Comment:
I'm not really sure how long this approach will be available for given it's [deprecated](https://github.com/google/closure-library/blob/master/closure/goog/net/cookies_deprecated.js#L23)

I can't really think of a nicer way of accessing the raw cookie for testing purposes short of spinning up an echo server on the same (and different) domains. Which is overkill. 

Looking at the Google Closure Library, they have extensive testing around `goog.net.Cookies` as is

Reply:
Yeah, we could probably leave this as is for now, but agree that closure testing is likely sufficient.